--- Makefile.orig	2017-08-23 14:07:38.562355000 +0000
+++ Makefile	2017-08-23 14:09:28.906845630 +0000
@@ -34,6 +34,11 @@
 GODOG_OPTS ?= ""
 PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
+ifeq ($(GO_BINDATA),)
+       GO_BINDATA_PATH := $(GOPATH)/bin/go-bindata
+else
+       GO_BINDATA_PATH := $(GO_BINDATA)
+endif
 
 # Linker flags
 VERSION_VARIABLES := -X $(REPOPATH)/pkg/version.minishiftVersion=$(MINISHIFT_VERSION) \
@@ -74,7 +79,7 @@
 vendor:
 	glide install -v
 
-$(ADDON_ASSET_FILE): $(GOPATH)/bin/go-bindata
+$(ADDON_ASSET_FILE): $(GO_BINDATA_PATH)
 	@mkdir -p $(ADDON_BINDATA_DIR)
 	go-bindata $(GO_BINDATA_DEBUG) -prefix $(ADDON_ASSETS) -o $(ADDON_ASSET_FILE) -pkg bindata $(ADDON_ASSETS)/...
 
