--- Makefile.orig	2017-05-12 22:21:30.038153237 +0000
+++ Makefile	2017-05-12 22:23:04.722010353 +0000
@@ -31,6 +31,12 @@
 GODOG_OPTS ?= ""
 PACKAGES := go list ./... | grep -v /vendor
 SOURCE_DIRS = cmd pkg test
+ifeq ($(GO_BINDATA),)
+       GO_BINDATA_PATH := $(GOPATH)/bin/go-bindata
+else
+       GO_BINDATA_PATH := $(GO_BINDATA)
+endif
+
 
 # Linker flags
 VERSION_VARIABLES := -X $(REPOPATH)/pkg/version.version=$(MINISHIFT_VERSION) \
@@ -55,7 +61,7 @@
 vendor:
 	glide install -v
 
-$(ADDON_ASSET_FILE): $(GOPATH)/bin/go-bindata vendor
+$(ADDON_ASSET_FILE): $(GO_BINDATA_PATH) vendor
 	@mkdir -p $(ADDON_BINDATA_DIR)
 	go-bindata $(GO_BINDATA_DEBUG) -prefix $(ADDON_ASSETS) -o $(ADDON_ASSET_FILE) -pkg bindata $(ADDON_ASSETS)/...
 
